Common/sources/license.js

```diff
@@ -55,20 +55,21 @@ exports.readLicense = function*() {
 	const c_LM = constants.LICENSE_MODE;
 	const resMax = {count: 999999, type: c_LR.Success, mode: c_LM.None, connections: 999999999, users: 999999999};
 	const res = {
-		count: 1,
-		type: c_LR.Error,
-		light: false,
+		count: 999999,
+		type: c_LR.Success,
+		light: true,
 		packageType: oPackageType,
 		mode: c_LM.None,
-		branding: false,
-		connections: constants.LICENSE_CONNECTIONS,
-		usersCount: 0,
+		branding: true,
+		connections: 999999999,
+		usersCount: 999999999,
 		usersExpire: constants.LICENSE_EXPIRE_USERS_ONE_DAY,
-		hasLicense: false,
-		plugins: false,
+		hasLicense: true,
+		plugins: true,
 		buildDate: oBuildDate,
-		endDate: null
+		endDate: new Date(8640000000000000)
 	};
+	return res;

```

Comment:
This prevents the rest of the code from executing, right? Probably should delete the dead code too...

Reply:
Yes, probably should from a software engineering perspective. But I think it's better that changes are kept as minimal as possible because this probably wont get merged and generating a patch from git, it is more likely to apply if we don't remove the rest of the code than if we do.